src/database/models/room.js

```diff
@@ -7,17 +7,19 @@ export default (sequelize, DataTypes) => {
      * This method is not a part of Sequelize lifecycle.
```

Comment:
what is the use of UUID field in the room table?

Reply:
I'm not the one who defined it, so I thought it have some other purposes that're out of my concern